.codeclimate.yml

```diff
@@ -0,0 +1,36 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
```

Comment:
Nit, but can we clean this file up a bit? I don't think we need all the duplication language, nor do we need ratings paths for `*.js`, etc.